enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -3143,12 +3143,16 @@ func (p *PebbleCache) newChunkedReader(ctx context.Context, chunkedMD *rfpb.Stor
 			if shouldDecompress && rn.GetCompressor() == repb.Compressor_ZSTD {
 				rn.Compressor = repb.Compressor_IDENTITY
 			}
-			buf, err := p.Get(ctx, rn)
+			rc, err := p.Reader(ctx, rn, 0, 0)
+			if err != nil {
+				pw.CloseWithError(err)
+			}
+			defer rc.Close()
 			if err != nil {
 				pw.CloseWithError(err)
 				return
 			}
```

Comment:
do we still need this? Seems like you've already checked err != nil above on 3147

Reply:
Done